docs/changelog/2024.2.0.rst

```diff
@@ -64,7 +64,7 @@ Trivial/Internal Changes
   in the |contributor guide|. (:pr:`2376`)
 - Separated the GitHub Action for checking hyperlinks in the
   documentation into its own GitHub Action. (:pr:`2392`)
-- Replaced |black| with |ruff| in the |pre-commit|
+- Replaced ``black`` with |ruff| in the |pre-commit|
```

Comment:
`` replaced | on just black?

Reply:
Correct. Since we switched our formatter from `black` to `ruff`, I removed the global reStructuredText substitution for `|black|`.